testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -396,24 +396,26 @@ def test_hes_error_estimation(self, ens1):
             "Unexpected standard deviation for bootstrapped samples in"
             " Harmonic Ensemble similarity"
         )
-        assert_almost_equal(average, expected_average, -2, err_msg)
-        assert_almost_equal(stdev, expected_stdev, -2, error_msg)
+        assert_allclose(average, expected_average, rtol=0, atol=1.5 * (1e2), err_msg=err_msg)
+        assert_allclose(stdev, expected_stdev, rtol=0, atol=1.5 * (1e2), err_msg=error_msg)
 
     def test_ces_error_estimation(self, ens1):
-        expected_average = 0.03
-        expected_stdev = 0.31
-        averages, stdevs = encore.ces([ens1, ens1],
-                                      estimate_error = True,
-                                      bootstrapping_samples=10,
-                                      clustering_method=encore.AffinityPropagationNative(preference=-2.0),
-                                      select="name CA and resnum 1-10")
-        average = averages[0,1]
-        stdev = stdevs[0,1]
-
-        assert_almost_equal(average, expected_average, decimal=1,
+            expected_average = 0.03
+            expected_stdev = 0.31
+            averages, stdevs = encore.ces([ens1, ens1],
+                                          estimate_error=True,
+                                          bootstrapping_samples=10,
+                                          clustering_method=encore.AffinityPropagationNative(preference=-2.0),
+                                          select="name CA and resnum 1-10")
+            average = averages[0, 1]
+            stdev = stdevs[0, 1]
+
+            assert_allclose(average, expected_average, rtol=0, atol=1.5*(1e-1),
                             err_msg="Unexpected average value for bootstrapped samples in Clustering Ensemble similarity")
-        assert_almost_equal(stdev, expected_stdev, decimal=0,
+            
+            assert_almost_equal(stdev, expected_stdev, decimal=0,
                             err_msg="Unexpected standard daviation  for bootstrapped samples in Clustering Ensemble similarity")
```

Comment:
for this assert_almost_equal statement 
the val of para. decimal = 0, 

which means that the abs(x,y) should be < 1.5 (which i found by running some examples after a few tests failed),
but for assert_allclose(x,y, rtol=0, atol=0) means the abs(x-y) = 0,

To resolve this should I go ahead and replace the assert_almost_equal which a if condition.
Please suggest some alternate ways to handle it.

Reply:
The equivalent for `decimal=0` is `atol=1.5e0` which is just `atol=1.5`:
```python
assert_allclose(stdev, expected_stdev, rtol=0, atol=1.5)
```
Does the code above work?

Reply:
yes

Reply:
Btw, can you please fix the spelling mistake: "daviation" --> "deviation"?